src/main/java/frc/robot/subsystems/grabber/Grabber.java

```diff
@@ -18,16 +21,26 @@ public class Grabber extends SubsystemBase {
   private final DigitalInput gamePieceSensor =
       new DigitalInput(RobotMap.GRABBER_GAME_PIECE_SENSOR_DIO);
 
+  private boolean runningCommand = false;
+
   public void spinMotors(double power) {
+    runningCommand = power==0.0?false:true;
```

Comment:
Checking floating point equality with zero is bad practice generally, maybe just check that abs(power) is less than some small very number